packages/editor/packages/history/src/components/HistoryItem.tsx

```diff
@@ -0,0 +1,47 @@
+import { Item, OnItemClick } from '../types';
+
+type Props = {
+	item: Item,
+	onClick: OnItemClick,
+	isCurrent: boolean,
+};
+
+const HistoryItem : React.VFC<Props> = ( props ) => {
+	return (
+		// eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+		<div
```

Comment:
Try button

Reply:
Breaks the style:


![image](https://user-images.githubusercontent.com/32631382/191417964-007a05bf-2cf3-4418-aa8f-d71311e802f8.png)